prow/config/config.yaml

```
@@ -1369,6 +1369,16 @@ branch-protection:
                   - 'idc-jenkins-ci-tiflash/build'
                   - 'license/cla'
                 strict: true
+            planner_refactory:
+              protect: true
+              required_status_checks:
+                contexts:
+                  - 'tide'
+                  - 'idc-jenkins-ci-tiflash/unit-test'
+                  - 'idc-jenkins-ci-tiflash/integration-test'
+                  - 'idc-jenkins-ci-tiflash/build'
+                  - 'license/cla'
```

Do we need to add `check-license` and `check-issue-triage-complete` here?
```suggestion
                contexts:
                  - 'tide'
                  - 'idc-jenkins-ci-tiflash/unit-test'
                  - 'idc-jenkins-ci-tiflash/integration-test'
                  - 'idc-jenkins-ci-tiflash/build'
                  - "check-issue-triage-complete"
                  - 'license/cla'
                  - "check-license"
```

---

It seems  `check-license` and `check-issue-triage-complete` only needed when brancn is `master`. other release-x.x branch also not include those two contexts

---

@Mini256 Do we need to add `check-license` and `check-issue-triage-complete` on feature branch?

---

Because feature branch will be merged directly to master, we should try to keep feature branch and master the same?

---

Because feature branch will be merged directly to master, we should try to keep feature branch and master the same?